app/components/product-walkthrough-feature-suggestion.hbs

```diff
@@ -10,14 +10,16 @@
     to learn how challenges work.
   </span>
 
-  <div class="absolute right-3 top-0 bottom-0 flex items-center justify-center">
-    <button
-      class="border border-blue-300 p-1 rounded hover:bg-blue-200"
-      type="button"
-      {{on "click" this.handleDismissButtonClick}}
-      data-test-dismiss-button
-    >
-      {{svg-jar "x" class="w-5 h-5 text-blue-500"}}
-    </button>
-  </div>
+  {{#if this.authenticator.isAuthenticated}}
```

Comment:
```suggestion
  {{#if this.isDismissable}}
```

We seem to have different conditions in the component - let's re-use the same here?